dataprofiler/tests/data_readers/test_avro_data.py

```diff
@@ -31,6 +32,16 @@ def setUpClass(cls):
             dict(path=os.path.join(test_dir, 'avro/snappy_compressed_intentionally_mislabeled_file.csv'), count=4),
         ]
 
+    def test_is_match_for_byte_streams(self):
+        """
+        Determine if the avro file can be automatically identified from byte stream
+        """
+        for input_file in self.input_file_names:
+            # as BytesIO Stream
+            byte_string = BytesIO(open(input_file['path'], 'rb').read())
```

Comment:
You use `with` statements with the opens in other tests but not this one, is there a reason they are different? I think we should make it consistent.